Header header
float32 drone_timeuint16 tag
uint16 size
int16 mx
int16 my
int16 mz
ardrone_autonomy::vector31 magneto_raw
ardrone_autonomy::vector31 magneto_rectified
ardrone_autonomy::vector31 magneto_offset
float32 heading_unwrapped
float32 heading_gyro_unwrapped
float32 heading_fusion_unwrapped
uint8 magneto_calibration_ok
uint32 magneto_state
float32 magneto_radius
float32 error_mean
float32 error_var
